Source/WTF/wtf/Compiler.h

```diff
@@ -641,3 +641,19 @@
 #else
 #define WTF_UNSAFE_BUFFER_USAGE
 #endif
+
+/* WTF_IGNORE_WARNINGS_IN_THIRD_PARTY_CODE */
+
+#define WTF_IGNORE_WARNINGS_IN_THIRD_PARTY_CODE_BEGIN \
+    WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN \
+    ALLOW_COMMA_BEGIN \
+    ALLOW_DEPRECATED_DECLARATIONS_BEGIN \
+    IGNORE_CLANG_WARNINGS_BEGIN("cast-align") \
+    IGNORE_CLANG_WARNINGS_BEGIN("thread-safety-reference-return")
+
+#define WTF_IGNORE_WARNINGS_IN_THIRD_PARTY_CODE_END \
+    WTF_ALLOW_UNSAFE_BUFFER_USAGE_END \
+    ALLOW_COMMA_END \
+    ALLOW_DEPRECATED_DECLARATIONS_END \
+    IGNORE_CLANG_WARNINGS_END \
+    IGNORE_CLANG_WARNINGS_END
```

Comment:
If you take my suggestion and use `IGNORE_CLANG_WARNINGS_BEGIN` and `IGNORE_WARNINGS_BEGIN` together, then the order of those two will matter, so:

```suggestion
#define WTF_IGNORE_WARNINGS_IN_THIRD_PARTY_CODE_END \
    IGNORE_CLANG_WARNINGS_END \
    IGNORE_WARNINGS_END \
    ALLOW_DEPRECATED_DECLARATIONS_END \
    ALLOW_COMMA_END \
    WTF_ALLOW_UNSAFE_BUFFER_USAGE_END \
```